lib/project_types/rails/commands/generate.rb

```diff
@@ -28,6 +28,7 @@ def self.extended_help
       end
 
       def self.run_generate(script, name, ctx)
+        Gem.gem_path(ctx)
```

Comment:
This method should probably be named better if we can just set the Gem path like this. Or alias it as `set_path`